features/currencyexchange/domain/build.gradle.kts

```diff
@@ -22,7 +22,9 @@ kotlin {
                 implementation(libs.kotlinx.coroutines.core)
                 implementation(libs.kotlinx.datetime)
                 implementation(projects.common.coroutinesKtx)
-                implementation(projects.common.domain)           }
+                implementation(projects.common.domain)
+                implementation(projects.features.currencyexchange.data.remote.mindera.rest.schema)
```

Comment:
Indentation

Reply:
[formatting](https://github.com/Mindera/Android-KMP-Template/pull/9/commits/b02611b1dda41dda34e9bbbfe36ba5faddb738bc) I hope I did it right